enums_test.go

```diff
@@ -21,7 +22,8 @@ func TestParseGlobalEnums(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(expected), string(b))
 	constsPath := "github.com/swaggo/swag/testdata/enums/consts"
-	assert.Equal(t, 64, p.packages.packages[constsPath].ConstTable["uintSize"].Value)
+	bitsizeHostUint := int(math.Round(math.Log2(float64(^uintptr(0)))))
+	assert.Equal(t, bitsizeHostUint, p.packages.packages[constsPath].ConstTable["uintSize"].Value)
```

Comment:
Can you try to reploce `bitsizeHostUint` with constant bits.UintSize from math/bits package?

Reply:
Yes, can confirm that works as well. Adjusted in e334757, tested on 8 different architectures through the [Alpine Linux CI](https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/55041)